### PR TITLE
Added some tests for ListView's 

### PR DESCRIPTION
**Pull request by limbera**

### CODE REVIEW

File: tests/test_rest_views.py [hunk 1]
```diff
@@ -5,6 +5,44 @@
 from .models import Poll
 
 
+class ListRestViewTest(TestCase):
+
+    def setUp(self):
+        self.c = Client()
```
Comment (funkybob) [inline tests/test_rest_views.py]: TestCase should already have self.client being a Client instance...

Comment (limbera) [inline tests/test_rest_views.py]: I was just copying what you dod in the class below.

Comment (funkybob) [inline tests/test_rest_views.py]: A good lesson in "don't copy without comprehension" :)

Comment (limbera) [inline tests/test_rest_views.py]: This is true. Shall I fix that class up too?

Comment (funkybob) [inline tests/test_rest_views.py]: Yes please.

Comment (limbera) [inline tests/test_rest_views.py]: Fixed.

**Pull request merged**